go.mod

```diff
@@ -212,3 +213,5 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/span
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter => ./exporter/googlecloudexporter
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/stackdriverexporter => ./exporter/stackdriverexporter
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver => ./receiver/kafkametricsreceiver
```

Comment:
Please try to keep things sorted.

Reply:
the existing replace statements aren't sorted, that's why I added this to the last line. I assumed that was the pattern.
I'll move this under the rest of the receivers.